LLama/Native/LLamaRopeType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace LLama.Native;
+
+public enum LLamaRopeType
```

Comment:
Please add comment for it.

Reply:
I think that's my responsibility, it's showing up here because this PR is based on #565